pulp_smash/pulp3/constants.py

```diff
@@ -20,6 +20,8 @@
 
 DISTRIBUTION_PATH = urljoin(BASE_PATH, 'distributions/')
 
+DOWNLOAD_POLICIES = ['immediate', 'on_demand', 'streamed']
```

Comment:
Did we rename `cahe-olny`?
```python
DOWNLOAD_POLICIES = ['cache_only', 'immediate', 'on_demand']
```
Constant on [Pulp file](https://github.com/pulp/pulp_file/blob/c7a90288e96d45faac191809cc9be8cca94a2117/pulp_file/tests/functional/constants.py#L12)

Reply:
I did so in another PR but we can do this in any order/way you recommend.

https://github.com/pulp/pulp_file/pull/140/files#diff-2d4578fd30e72ae31e7baef653596f7fL12

Reply:
https://github.com/PulpQE/pulp-smash/issues/1157